### ISSUE TITLE
The widget doesn't center vertically

### ISSUE THREAD

**Issue by sudhirkhanger** [kotelnik/plasma-applet-thermal-monitor#36]
![screenshot_20180105_104406](https://user-images.githubusercontent.com/841656/34596468-eb558d5a-f205-11e7-87b9-38c1796e9d4e.png)

As you can see in the image above, the temperature is not centered vertically. It would be really great if the widget can show the single temperature centered vertically. You can see how Event Calendar widget centers vertically nicely irrespective of its scale.

Thanks.

**Comment by andreyorst** [comment]
Yeah, I can confirm that this happens still. I've already opened same issue #30, but i think that @kotelnik misunderstood me
